runtime/memory.x

```diff
@@ -2,7 +2,7 @@
 
 ROM_ORG   = 0x00000000;
 META_ORG  = ROM_SIZE;
-ICCM_ORG  = 0x40002000;
+ICCM_ORG  = 0x40004000;
```

Comment:
In the ROM linker script, ICCM is at `0x40000000`. https://github.com/chipsalliance/caliptra-sw/blob/main/rom/dev/src/rom.ld

Why is it different here?

Reply:
FMC is loaded at 0x40000000 and RT load addr is 0x40004000

Reply:
Got it, thanks!